### ISSUE TITLE
Select Text / Copy part of a note [Feature Request]

### ISSUE THREAD

**Issue by alltheseas** [vitorpamplona/amethyst#57]
## User Story ##
As a pleb Amethyst user who would like to copy a particular bit of information from a note, I would like to explicitly select and copy part of the note (not the whole note), so I can save/share only the relevant part of the note.

### Acceptance Criteria ###
1. Pleb can select part of a note,
2. Pleb can copy selected text

![image](https://user-images.githubusercontent.com/64376233/215944384-2cf579eb-2d32-4469-851e-54d18a814466.png)

**Comment by vitorpamplona** [comment]
Consolidating: whole-note "Copy Text" exists in the note quick-action menu, and partial text selection is tracked by the accepted request #1373 (blocked on the new parser/renderer). Closing this one as a duplicate of #1373.

---
_Generated by [Claude Code](https://claude.ai/code)_
